serverless/pages/saved-objects.mdx

```diff
@@ -85,3 +85,16 @@ TBD: Are these settings configurable in serverless?
 The <DocLink id="enKibanaSettings" section="savedObjects-maxImportExportSize">`savedObjects.maxImportExportSize`</DocLink> configuration setting limits the number of saved objects that you can export.
 </DocCallOut> */}
 
+## Copy to other spaces
+
+Copy saved objects and their related objects between spaces.
+
+1. Click the actions icon image:images/actions_icon.png[Actions icon].
```

Comment:
I see text rather than the image (but I'm not sure if that's normal for the preview)

Reply:
LGTM otherwise!

Reply:
You're right, it's not normal, I kept the asciidoc notation -_-

Reply:
Looks right now!